<?xml version="1.0" encoding="UTF-8"?><testrun name="BlackJackTest" project="BlackJackexp" tests="17" started="17" failures="1" errors="0" ignored="0">
  <testsuite name="com.QA.BlackJackexp.BlackJackTest" time="0.004">
    <testcase name="testy10" classname="com.QA.BlackJackexp.BlackJackTest" time="0.0"/>
    <testcase name="testy11" classname="com.QA.BlackJackexp.BlackJackTest" time="0.0"/>
    <testcase name="testy12" classname="com.QA.BlackJackexp.BlackJackTest" time="0.0"/>
    <testcase name="testy13" classname="com.QA.BlackJackexp.BlackJackTest" time="0.0"/>
    <testcase name="testy14" classname="com.QA.BlackJackexp.BlackJackTest" time="0.0"/>
    <testcase name="testy15" classname="com.QA.BlackJackexp.BlackJackTest" time="0.0"/>
    <testcase name="testy16" classname="com.QA.BlackJackexp.BlackJackTest" time="0.0"/>
    <testcase name="testy1" classname="com.QA.BlackJackexp.BlackJackTest" time="0.0"/>
    <testcase name="testy2" classname="com.QA.BlackJackexp.BlackJackTest" time="0.0"/>
    <testcase name="testy3" classname="com.QA.BlackJackexp.BlackJackTest" time="0.0"/>
    <testcase name="testy4" classname="com.QA.BlackJackexp.BlackJackTest" time="0.0"/>
    <testcase name="testy5" classname="com.QA.BlackJackexp.BlackJackTest" time="0.0"/>
    <testcase name="testy6" classname="com.QA.BlackJackexp.BlackJackTest" time="0.002">
      <failure>java.lang.AssertionError: expected:&lt;20&gt; but was:&lt;0&gt;&#13;
	at org.junit.Assert.fail(Assert.java:88)&#13;
	at org.junit.Assert.failNotEquals(Assert.java:834)&#13;
	at org.junit.Assert.assertEquals(Assert.java:645)&#13;
	at org.junit.Assert.assertEquals(Assert.java:631)&#13;
	at com.QA.BlackJackexp.BlackJackTest.testy6(BlackJackTest.java:34)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testy7" classname="com.QA.BlackJackexp.BlackJackTest" time="0.0"/>
    <testcase name="testy8" classname="com.QA.BlackJackexp.BlackJackTest" time="0.001"/>
    <testcase name="testy9" classname="com.QA.BlackJackexp.BlackJackTest" time="0.001"/>
    <testcase name="testy" classname="com.QA.BlackJackexp.BlackJackTest" time="0.0"/>
  </testsuite>
</testrun>
